api/pkg/database/size_enforcer.go

```diff
@@ -0,0 +1,119 @@
+package database
+
+import (
+	"fmt"
+	"github.com/fsnotify/fsnotify"
+	"github.com/up9inc/mizu/shared"
+	"github.com/up9inc/mizu/shared/debounce"
+	"log"
+	"mizuserver/pkg/models"
+	"os"
+	"strconv"
+	"time"
+)
+
+const percentageOfMaxSizeBytesToPrune = 15
+const defaultMaxDatabaseSizeBytes = 200 * 1000 * 1000
+
+func StartEnforcingDatabaseSize() {
+	watcher, err := fsnotify.NewWatcher()
+	if err != nil {
+		log.Printf("Error creating filesystem watcher for db size enforcement: %v\n", err) // TODO: make fatal
```

Comment:
Jira ticket instead of TODO.

Reply:
changed to fatal as this no longer needs to be immediately demoed